packages/common-server/src/template.ts

```diff
@@ -31,8 +31,16 @@ function copyTemplateProps({
 }: TemplateFunctionProps) {
   const tempNoteProps = _.pick(templateNote, TemplateUtils.TEMPLATE_COPY_PROPS);
   _.forEach(tempNoteProps, (v, k) => {
-    // @ts-ignore
-    targetNote[k] = v;
+    if (k === "custom" && v) {
+      if (targetNote.custom === undefined) targetNote.custom = {};
+      Object.keys(v).forEach((key) => {
+        // @ts-ignore
+        targetNote[k][key] = targetNote[k][key] || v[key];
```

Comment:
instead of using `targetNote[k]` i would use `targetNote['custom']` to make this a little easier to read